compose/Makefile

```diff
@@ -1,9 +1,9 @@
 kong-postgres:
-	COMPOSE_PROFILES=database KONG_DATABASE=postgres docker-compose up -d
+	COMPOSE_PROFILES=database KONG_DATABASE=postgres docker-compose up -d || docker compose up -d
```

Comment:
Please replace `docker-compose` by `docker compose` instead of using both. The new version of docker has been out for enough time now.

Also, please add a mention in the readme about the minimum Docker version supported when doing this change.